### PR TITLE
Make updates to 9280 specific

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

File: draft-rswg-rfc7990-updates.mkd [hunk 1]
```diff
@@ -96,8 +96,24 @@ and by dropping the requirement that the XML be embedded in the PDF publication
...
-{{updating}} and {{pub-versions}} in this document update that policy.
+## Changes to RFC 9280 {#changes-to-9280}
+
+Section 7.6 of {{RFC9280}} currently says:
+
+> Once published, RFC Series documents are not changed.
+
+This document replaces that sentence with:
+
+> Once published, RFC Series documents may be re-issued, but unintended changes shall be avoided, and content semantics are to be preserved.
```
Comment (martinthomson) [inline draft-rswg-rfc7990-updates.mkd:107]: Do you really think that "unintended changes shall be avoided" is worth saying?

```suggestion
> Once published, RFC Series documents may be re-issued, but the semantic content of published documents shall be preserved.
```

File: draft-rswg-rfc7990-updates.mkd [hunk 2]
```diff
@@ -96,8 +96,24 @@ and by dropping the requirement that the XML be embedded in the PDF publication
...
+
+This document also creates a new policy that would exist in Section 7 of {{RFC9280}}:
+
+> 7.8.  Consistency
+> 
+> The series as a whole is consistently presented.
+RFCs are copyedited, formatted, published, and may be reissued so that they are consistently presented to the public.
```
Comment (martinthomson) [inline draft-rswg-rfc7990-updates.mkd:114]:
```suggestion
RFCs are copyedited, formatted, published, and may be reissued to maintain a consistent presentation across the Series.
```

File: draft-rswg-rfc7990-updates.mkd (hunk 1 above)
Comment (paulehoffman) [reply draft-rswg-rfc7990-updates.mkd:107]: I agree with Martin here. Unintended everything is always avoided.

File: draft-rswg-rfc7990-updates.mkd (hunk 2 above)
Comment (paulehoffman) [reply draft-rswg-rfc7990-updates.mkd:114]: Not sure on this one.

File: draft-rswg-rfc7990-updates.mkd (hunk 1 above)
Comment (elear) [reply draft-rswg-rfc7990-updates.mkd:107]: That's fair.

File: draft-rswg-rfc7990-updates.mkd (hunk 2 above)
Comment (elear) [reply draft-rswg-rfc7990-updates.mkd:114]: That's fine as well.

**Comment by paulehoffman** [comment]
Sent to the mailing list 2024=03-05

File: draft-rswg-rfc7990-updates.mkd [hunk 3]
```diff
@@ -96,8 +96,24 @@ and by dropping the requirement that the XML be embedded in the PDF publication
 
 Historical text from {{RFC7990}} such as Section 2 ("Problem Statement"), Section 4 ("Overview of the Decision-Making Process"), and Section 10 ("Transition Plan") are not copied to this document.
 
-Section 7.6 of "RFC Editor Model (Version 3)" {{RFC9280}} says "Once published, RFC Series documents are not changed."
-{{updating}} and {{pub-versions}} in this document update that policy.
+## Changes to RFC 9280 {#changes-to-9280}
+
+Section 7.6 of {{RFC9280}} currently says:
```
Comment (cabo) [inline draft-rswg-rfc7990-updates.mkd:101]: Is the "currently" anticipaion that we'll change the content as well?

Comment (paulehoffman) [reply draft-rswg-rfc7990-updates.mkd:101]: Please send such comments to the RSWG mailing list. Thanks!

**Pull request merged**